internal/pkg/ccloudv2/schema-registry.go

```diff
@@ -0,0 +1,52 @@
+package ccloudv2
+
+import (
+	"context"
+	"net/http"
+
+	srcm "github.com/confluentinc/ccloud-sdk-go-v2/srcm/v2"
+)
+
+func newSchemaRegistryClient(url, userAgent string, unsafeTrace bool) *srcm.APIClient {
+	cfg := srcm.NewConfiguration()
+	cfg.Debug = unsafeTrace
+	cfg.HTTPClient = newRetryableHttpClient(unsafeTrace)
+	cfg.Servers = srcm.ServerConfigurations{{URL: url}}
+	cfg.UserAgent = userAgent
+
+	return srcm.NewAPIClient(cfg)
+}
+
+func (c *Client) SchemaRegistryApiContext() context.Context {
+	return context.WithValue(context.Background(), srcm.ContextAccessToken, c.AuthToken)
+}
+
+func (c *Client) GetSchemaRegistryClusterByEnvironment(environment string) (srcm.SrcmV2ClusterList, *http.Response, error) {
+	return c.SchemaRegistryClient.ClustersSrcmV2Api.ListSrcmV2Clusters(c.SchemaRegistryApiContext()).Environment(environment).Execute()
```

Comment:
This should set the page size to `ccloudV2ListPageSize`, and probably handle next page tokens, to avoid issues with users going over the default page size limit and getting truncated results.